vendors/twtg/drivers/neon-vs-qt-xx-xx/package.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "neon-vs-qt-xx-xx",
```

```suggestion
  "name": "neon-vs-qt",
```

Same counts for the folder name